components/resolved_ts/src/metrics.rs

```diff
@@ -66,22 +66,37 @@ lazy_static! {
     .unwrap();
     pub static ref RTS_MIN_RESOLVED_TS: IntGauge = register_int_gauge!(
         "tikv_resolved_ts_min_resolved_ts",
-        "The minimal (non-zero) resolved ts for observe regions"
+        "The minimal (non-zero) resolved ts for observed regions"
+    )
+    .unwrap();
+    pub static ref RTS_MIN_SAFE_TS_REGION: IntGauge = register_int_gauge!(
+        "tikv_resolved_ts_min_safe_ts_region",
+        "The region which has minimal safe ts"
+    )
+    .unwrap();
+    pub static ref RTS_MIN_SAFE_TS: IntGauge = register_int_gauge!(
+        "tikv_resolved_ts_min_safe_ts",
```

Comment:
Is this not used yet in grafana dashboard?

Reply:
Yes. RTS_MIN_RESOLVED_TS is not used either. I'm making them equal.